### PR TITLE
Add docstring for `genlist` function and installation/usage instructions in README.md

### PR DESCRIPTION
**Pull request by VarLad**

### CODE REVIEW

**Comment by VarLad** [comment]
@Roger-luo I think I should've deleted the previous branch and forked it again. 
If its an issue, do tell. 

Otherwise, its good to merge. :crossed_fingers:

File: src/YaoPastaQ.jl [hunk 1]
```diff
@@ -5,6 +5,9 @@ export genlist, apply!, PastaQReg
 flblock(blk::AbstractBlock) = YaoBlocks.Optimise.simplify(blk, rules=[YaoBlocks.Optimise.to_basictypes])
 sublocs(subs, locs) = [locs[i] for i in subs]   
 
+"""
+Returns the PastaQ input list from Yao's QBIR
```
Comment (Roger-luo) [inline src/YaoPastaQ.jl:9]: You need to put function signature in the docstring

File: README.md [hunk 2]
```diff
@@ -4,3 +4,25 @@
 [![Dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://QuantumBFS.github.io/YaoPastaQ.jl/dev)
 [![Build Status](https://github.com/QuantumBFS/YaoPastaQ.jl/workflows/CI/badge.svg)](https://github.com/QuantumBFS/YaoPastaQ.jl/actions)
 [![Coverage](https://codecov.io/gh/QuantumBFS/YaoPastaQ.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/QuantumBFS/YaoPastaQ.jl)
+
+To install, in the Julia REPL do
+
+```julia
+julia> add https://github.com/QuantumBFS/YaoPastaQ.jl
```
Comment (Roger-luo) [inline README.md:11]: Can you just use the README template in Configurations.jl? This would be more consistent

**Review by Roger-luo — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (VarLad) [reply README.md:11]: Thanks for pointing it out, will do both as soon as possible (after I fetch some sleep) 😃

**Comment by VarLad** [comment]
Sorry, I'm aware that I should've done this in 2 PRs, but it wasn't much so I just stuffed the new gates and fixes in here :grimacing:

**Review by Roger-luo — APPROVED** [review]
LGTM

**Pull request merged**